--- graphene-0.99.2/src/graphene-point3d.c.orig	2014-06-23 12:40:39.000000000 +0200
+++ graphene-0.99.2/src/graphene-point3d.c	2014-07-27 08:09:01.524038179 +0200
@@ -37,6 +37,7 @@
 #include "config.h"
 
 #include <math.h>
+#include <stdlib.h>
 
 #include "graphene-point3d.h"
 #include "graphene-vec3.h"
--- graphene-0.99.2/src/graphene-point.c.orig	2014-06-23 12:40:39.000000000 +0200
+++ graphene-0.99.2/src/graphene-point.c	2014-07-27 08:08:36.847372544 +0200
@@ -36,6 +36,7 @@
 #include "config.h"
 
 #include <math.h>
+#include <stdlib.h>
 
 #include "graphene-point.h"
 #include "graphene-simd4f.h"
--- graphene-0.99.2/src/graphene-quad.c.orig	2014-06-23 12:40:39.000000000 +0200
+++ graphene-0.99.2/src/graphene-quad.c	2014-07-27 08:09:21.187370685 +0200
@@ -35,6 +35,8 @@
 #include "graphene-rect.h"
 #include "graphene-line-segment.h"
 
+#include <stdlib.h>
+
 /**
  * graphene_quad_alloc: (constructor)
  *
--- graphene-0.99.2/src/graphene-quaternion.c.orig	2014-06-23 12:40:39.000000000 +0200
+++ graphene-0.99.2/src/graphene-quaternion.c	2014-07-27 08:09:35.794036737 +0200
@@ -35,6 +35,7 @@
 #include "config.h"
 
 #include <math.h>
+#include <stdlib.h>
 
 #include "graphene-quaternion.h"
 
--- graphene-0.99.2/src/graphene-rect.c.orig	2014-06-23 12:40:39.000000000 +0200
+++ graphene-0.99.2/src/graphene-rect.c	2014-07-27 08:09:54.334035960 +0200
@@ -37,6 +37,7 @@
 #include "config.h"
 
 #include <math.h>
+#include <stdlib.h>
 
 #include "graphene-rect.h"
 
--- graphene-0.99.2/src/graphene-simd4f.h.orig	2014-06-23 12:40:39.000000000 +0200
+++ graphene-0.99.2/src/graphene-simd4f.h	2014-07-27 08:10:55.077366743 +0200
@@ -586,7 +586,7 @@ typedef union {
                           __ua.f[3] < __ub.f[3] ? __ua.f[3] : __ub.f[3]); \
   }))
 
-# define graphene_simd4f_max (a,b) \
+# define graphene_simd4f_max(a,b) \
   (G_GNUC_EXTENSION ({ \
     graphene_simd4f_union_t __ua = { (a) }; \
     graphene_simd4f_union_t __ub = { (b) }; \
--- graphene-0.99.2/src/graphene-size.c.orig	2014-06-23 12:40:39.000000000 +0200
+++ graphene-0.99.2/src/graphene-size.c	2014-07-27 08:11:34.260698434 +0200
@@ -33,6 +33,7 @@
 #include "config.h"
 
 #include <math.h>
+#include <stdlib.h>
 
 #include "graphene-point.h"
 
