--- graphene-1.8.2/meson.build.orig	2018-06-13 18:51:25.000000000 +0200
+++ graphene-1.8.2/meson.build	2018-10-22 05:51:10.943800526 +0200
@@ -216,7 +216,7 @@
 
 # Optional dependency on GObject-Introspection; if GObject is disabled
 # then we don't build introspection data either
-build_gir = build_gobject and get_option('introspection') and get_option('default_library') == 'shared'
+build_gir = build_gobject and get_option('introspection') and get_option('default_library') != 'static'
 
 # Check for InitOnce on Windows
 if host_system == 'windows'
